hank-core/src/main/java/com/liveramp/hank/zookeeper/ZooKeeperPlus.java

```diff
@@ -375,12 +375,26 @@ public void ensureCreated(String path, byte[] value) throws InterruptedException
     }
 
     public void ensureCreated(String path, byte[] value, CreateMode createMode) throws InterruptedException, KeeperException {
-      if (exists(path, false) == null) {
+      if (!path.isEmpty() && exists(path, false) == null) {
+        ensureCreated(getParent(path), null, createMode);
```

Comment:
We want `value` to be null for parent nodes? I'd believe it -- just wanted to check :)